.github/workflows/main.yml

```diff
@@ -41,6 +41,7 @@ jobs:
       # See: https://github.com/actions/setup-node/issues/529
       - run: npm i -g npm@9
       - run: npm ci
+      - run: npx playwright install --with-deps
```

Comment:
I think I've gotten guidance to avoid using `npx` (in favor of creating new commands to invoke in package.json, like a `playwright-install-deps` command), but I don't remember the full justification. Can wait and see if @rajsite responds when he reviews.